PROJECTINFO.cmake

```diff
@@ -0,0 +1,15 @@
+##### PROJECTINFO for CMake
+
+SET( BASELIB		"yui" )		# don't change this
```

Comment:
Is this still correct now that it's no longer of the big thing that included everything?

Reply:
I do not know that Cmake magic, why it's there. But that's the same in all UI plugins, even in the normal Qt ([here](https://github.com/libyui/libyui-qt/blob/master/PROJECTINFO.cmake#L3)) or Ncurses ([here](https://github.com/libyui/libyui-ncurses/blob/master/PROJECTINFO.cmake#L3)). It was just copy&paste...

Reply:
OK then.